models/getArticlesModel.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection");
+
+async function getArticlesModels() {
+  try {
+    const result = await db.query(`SELECT * FROM articles `);
+
+    const article = result.rows;
+    return article;
+  } catch (error) {
+    throw error;
+  }
+}
+
+async function getArticleById(id) {
+  try {
+    const result = await db.query(
+      `SELECT * FROM articles WHERE article_id = $1`,
+      [id]
+    );
+
+    const article = result.rows[0];
+    return article;
+  } catch (error) {
+    throw error;
```

Comment:
I would use Promise.reject - you've got two catch blocks because you also have one in your controller as well. You should be doing the catching in the controller, not here

Reply:
Done